### ISSUE TITLE
allow commas in clean-currency

### ISSUE THREAD

**Issue by dpetrovics** [racehub/util#2]
allow commas in the amounts by stripping out commas before the parsing.

**Comment by dpetrovics** [comment]
```
racehub.models.payment.price-calculation> (u/clean-currency "1")
"1.00"
racehub.models.payment.price-calculation> (u/clean-currency "10")
"10.00"
racehub.models.payment.price-calculation> (u/clean-currency "1000")
"1,000.00"
racehub.models.payment.price-calculation> (u/clean-currency "1,000")
"0.00"
```

**Comment by sritchie** [comment]
Fixed by #26
